### ISSUE TITLE
Deprecate Thread Ratings in 1.9

### ISSUE THREAD

**Issue by Ben-MyBB** [mybb/mybb#3498]
Just following up on the vote we took on this at: https://community.mybb.com/thread-220554.html

The vote was to remove these in 1.9 so this issue is to track that.

**Comment by effone** [comment]
(lol @justinsoltesz `Depreciate` is also a correct word and a synonym of `deprecate`).

**Comment by justinsoltesz** [comment]
@effone Depreciate and deprecate have different meanings.

https://stackoverflow.com/questions/9208091/the-difference-between-deprecated-depreciated-and-obsolete/9208164

**Comment by justinsoltesz** [comment]
Also, on topic, I believe the plan that has been discussed for thread ratings in 1.9 is to remove them from the templates (which is essentially already the case in the new theme) but not to remove the ratings that have already been stored in the database. This will allow a plugin to use that existing data if desired.

**Comment by kawaii** [comment]
^ and keeping that data will also allow for someone (me) to create the plugin which reinstates thread ratings to 1.9. :wink:

**Comment by euantorano** [comment]
> Also, on topic, I believe the plan that has been discussed for thread ratings in 1.9 is to remove them from the templates (which is essentially already the case in the new theme) but not to remove the ratings that have already been stored in the database. This will allow a plugin to use that existing data if desired.

Yes, that is the plan exactly.

**Comment by yuliu** [comment]
Current template shows this field and there's no style for correctly displaying it, just raw list items.

**Comment by justinsoltesz** [comment]
> Current template shows this field and there's no style for correctly displaying it, just raw list items.

Correct, there is no style because thread ratings are being deprecated but the template is displaying it because the work to deprecate thread ratings hasn't been finished yet.
